frontend/express/public/core/user-management/javascripts/countly.views.js

```diff
@@ -0,0 +1,85 @@
+/* global countlyVue, countlyAuth, CV, app, countlyCommon */
+
+// create vue view
+var ManageUsersView = countlyVue.views.create({
+    template: CV.T("/core/user-management/templates/manage-users.html"),
+    components: {},
+    data: function() {
+        return {
+            isOpened: false,
+            constants: {
+                "visibilityOptions": [
+                    {label: "Global", value: "global", description: "Can be seen by everyone."},
+                    {label: "Private", value: "private", description: "Can be seen by the creator."}
+                ],
+                "availableProps": [
+                    {label: "Type 1", value: 1},
+                    {label: "Type 2", value: 2},
+                    {label: "Type 3", value: 3}
+                ]
```

Comment:
Lots of familiar things from vue-example plugin. Are you really sure about that you want them to be merged? 😄

Reply:
nice catch but this PR deprecated. :D I thought I closed this.
current one is this: https://github.com/Countly/countly-server/pull/1897